### PR TITLE
minor cleanup to some pipelines notebooks

### PR DESCRIPTION
**Pull request by amygdala**

### CODE REVIEW

File: ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_images.ipynb [hunk 1]
```diff
@@ -592,16 +592,16 @@
       "outputs": [],
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_images.ipynb:592]: <pre class="ql-syntax" spellcheck="false">Line #2.    def pipeline(project: str = PROJECT_ID):
</pre><p>You can use a variable as the default?</p><p> </p><p>Didn't know that was possible and even if possible, I haven't seen this used often.</p><p>Consider just removing the default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (ivanmkc) [inline ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_images.ipynb:592]: <pre class="ql-syntax" spellcheck="false">Line #2.        "image_classif_pipeline.json",
</pre><p>Is 'classif' a typo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

File: ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_tabular.ipynb [hunk 2]
```diff
@@ -604,13 +604,14 @@
       "outputs": [],
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_tabular.ipynb:604]: <pre class="ql-syntax" spellcheck="false">Line #2.    def pipeline(project: str = PROJECT_ID):
</pre><p>ditto</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

File: ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_text.ipynb [hunk 3]
```diff
@@ -695,7 +695,9 @@
       "outputs": [],
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_text.ipynb:695]: <pre class="ql-syntax" spellcheck="false">Line #2.        "text_classsif_pipeline.json",
</pre><p>Typo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

**Review by ivanmkc — APPROVED** [review]
LGTM

File: ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_images.ipynb (hunk 1 above)
Comment (amygdala) [reply ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_images.ipynb:592]: well, it was meant to be just an abbreviation of 'classification' to make the file name shorter :) but if it seems weird maybe I should change it.

File: ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_text.ipynb (hunk 3 above)
Comment (amygdala) [reply ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_text.ipynb:695]: same as above... it was meant to be just an abbreviation of 'classification' -- if it struck you as weird maybe that's a cue to change it.

File: ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_images.ipynb (hunk 1 above)
Comment (amygdala) [reply ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_images.ipynb:592]: It works well, and is a useful idiom for running pipelines in notebooks.  Hmm, let me give this some thought.

**Comment by amygdala** [comment]
Thanks, Ivan.  I'm going to go ahead and pull it in as is (there's one change in particular I want to get in), but let me give some thought to your suggestions.

**Pull request merged**